tencentcloud/provider.go

```diff
@@ -412,6 +412,15 @@ SSL Certificates
     tencentcloud_ssl_certificate
     tencentcloud_ssl_pay_certificate
 
+SSM
+  Data Source
+    tencentcloud_ssm_secrets
+	tencentcloud_ssm_secret_versions
```

Comment:
if secret version can be all listed with data source ssm secrets. do not need data source secret versions